.github/workflows/windows_builds.yml

```diff
@@ -49,6 +49,28 @@ jobs:
       - name: Setup python and scons
         uses: ./.github/actions/godot-deps
 
+      - name: Download pre-built DirectX Shader Compiler
+        uses: dsaltares/fetch-gh-release-asset@1.0.0
+        with:
+          repo: Microsoft/DirectXShaderCompiler
+          version: tags/v1.7.2207
+          file: dxc_2022_07_18.zip
```

Comment:
We're using a newer version in the PR's OP (`2023_03_01`), might be worth updating this too. Though if the CI artifacts are tested as working, this shouldn't be blocking for this PR, it's actually nice to know multiple versions of dxc are working with the same codebase.

And we should assess latest dxc too, the current docs in https://github.com/godotengine/godot-docs/pull/8610 don't actually specify a version, so users will install the latest release.

Reply:
Actually, we're only using the validation and signing part, that probably doesn't change much between releases.

I've pushed with the latest DXC, tested.